ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -29,22 +29,29 @@
 #include <wiredtiger.h>
 #include <wiredtiger_ext.h>
 
-#include <aws/core/Aws.h>
 #include "s3_connection.h"
+#include "s3_log_system.h"
+#include <aws/core/Aws.h>
+#include <aws/core/utils/memory/stl/AWSString.h>
+#include <aws/core/utils/logging/DefaultLogSystem.h>
+#include <aws/core/utils/logging/AWSLogging.h>
 
 #define UNUSED(x) (void)(x)
 
 /* S3 storage source structure. */
 typedef struct {
     WT_STORAGE_SOURCE storageSource; /* Must come first */
     WT_EXTENSION_API *wtApi;         /* Extension API */
+    uint32_t awsVerboseLevel;
+
 } S3_STORAGE;
 
 typedef struct {
     /* Must come first - this is the interface for the file system we are implementing. */
     WT_FILE_SYSTEM fileSystem;
     S3_STORAGE *s3Storage;
```

Comment:
I know this isn't part of the PR, but we should be somewhat consistent with variable names, particularly inside a `struct`. For example,
``` 
typedef struct {
    WT_FILE_SYSTEM fileSystem;
    S3_STORAGE *storage;
    S3Connection *connection;
    S3LogSystem *log;
} S3_FILE_SYSTEM;
```